pages/board.tsx

```diff
@@ -152,23 +159,90 @@ export default class BoardPage extends React.Component<Props, State> {
     );
   }
 
+  _renderBoard(data: Board) {
+    return (
+      <MoveCardMutation mutation={MOVE_CARD}>
+        {_moveCard => {
```

Comment:
should rename `_moveCard` to `moveCard`.  prefixing with an underscore is just a way of telling tslint to ignore the variable / parameter.